wcfsetup/install/files/lib/system/database/table/DatabaseTable.class.php

```diff
@@ -157,30 +156,25 @@ public function getIndices()
      *
      * @return  string      database table name
      */
-    public function getName()
+    public function getName(): string
     {
         return $this->name;
     }
 
     /**
      * Returns a `DatabaseTable` object with the given name.
      *
-     * @param string $tableName
      * @return  static
      */
-    public static function create($tableName)
+    public static function create(string $tableName)
```

Comment:
```suggestion
    public static function create(string $tableName): static
```
(and update the PHPDoc)